### ISSUE TITLE
Add codecov

### ISSUE THREAD

**Issue by julianstanley** [miraep8/rendseq#6]
Once we have tests setup (depends on #2) we should set up codecov to keep track of what lines of the package we've tested.

**Comment by miraep8** [comment]
Sure, this is another service I am unfamiliar with but open to learning more/incorporating it!

**Comment by miraep8** [comment]
If its ok with you - I am going to take a go at implementing this! :) I am just interested in learning a bit more about codecov/think it would be useful for some of my other repos as well! :

**Comment by julianstanley** [comment]
Sure thing!

I setup coveralls.io on my fork on Saturday--which is even simpler if you want to use that. 

All you have to do is make a coveralls account, link it to the repo, then you generate a key that you put in your .coveralls.yml file:

```
$ cat .coveralls.yml
repo_token: "ATI6AePdxwQpcCA8sod69GsONDKU"
```

Then when you're ready to generate coverage based on tests, you can run the coverage package:

```
$  python3 -m coverage run -m pytest tests/
```

Then you just run the coveralls command and it'll submit it to coveralls online and send a link

```
$ coveralls
```

If you want to go ahead and make a coveralls account and then commit a `.coveralls.yml` to master, then I can start submitting my tests coverage to your coveralls account. 

Then with #5 you can just add the coverage and coveralls commands to the CI build

**Comment by miraep8** [comment]
Great!  Thanks for the info.  I am planning to add the codecov to the CI build, should also be coming later this week. :)

**Comment by miraep8** [comment]
Closed by #26
